slidez-fe-core/src/App.tsx

```diff
@@ -14,4 +18,21 @@ function App() {
   )
 }
```

Comment:
I think, you can drop it

Reply:
Am I getting this correct: I need to change my css to scss, refactor weird icon thing, move variables to global css file and delete stuff that was created by "create-react-appication" command?